src/styles/components/_page-header.scss

```diff
@@ -30,6 +30,10 @@
 
 .w-page-header__headline {
   margin: 0 auto 11px;
+
+  &.w-code {
```

Comment:
I think we updated code inside of headers on master to look nicer. I'm not 100% sure if you'll still need this if we rebase against master.

Reply:
I'll let you deal :)

DevSite makes the 404 page appear in code font. I like it as a stylistic thing.